easylogger/src/elog_buf.c

```diff
@@ -82,14 +82,16 @@ void elog_buf_output(const char *log, size_t size) {
  * flush all buffered logs to output device
  */
 void elog_flush(void) {
-    /* lock output */
-    elog_output_lock();
-    /* output log */
-    elog_port_output(log_buf, buf_write_size);
-    /* reset write index */
-    buf_write_size = 0;
-    /* unlock output */
-    elog_output_unlock();
+    if(buf_write_size){
```

Comment:
```suggestion
    if(buf_write_size)
        return;
```

直接在最前面return即可，不需要把原有逻辑都包起来

Reply:
代码已更新